components/ContactForm.js

```diff
@@ -0,0 +1,34 @@
+var ContactForm = React.createClass({
+  propTypes: {
+    contact: React.PropTypes.object.isRequired
+  },
+
+  render: function() {
+    return (
+      <form className = {'contactForm'} >
+      <input  type = {'text'}  placeholder = {'Imię'}>
+          value: this.props.contact.firstName,
+      </input>
+        }),
+        
+     
+      <input  type = {'text'}  placeholder = {'Nazwisko'}>
+          value: this.props.contact.lastName,
+      </input>
+        }),
+    
+   
+      <input  type = {'email'}  placeholder = {'Email'}>
+          value: this.props.contact.email,
+      </input>
+        }),
+       
+        
+        
+        
+       <button type = {'submit'} "Dodaj kontakt" />
```

Comment:
`"Dodaj kontakt"` - this is an value of attribute value for button - same as in html